addons/mail/static/src/core/common/mail_fullscreen.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.Fullscreen">
+        <div class="fixed-top vw-100 vh-100" t-if="fullscreen.component">
+            <t t-component="fullscreen.component" t-props="fullscreen.props"/>
+        </div>
```

Comment:
Nitpick: prefer put `t-if` as 1st attribute (easier to see this is a conditional node)